markdown/dev/tutorials/getting-started-vercel/en.md

```diff
@@ -146,9 +146,10 @@ The Root Directory setting will determine the build type for the project.
 1. Change the name of the project, if you wish.
 Names can consist of alphanumeric lowercase and hyphen characters.
 2. Change the Root Directory to the desired setting, as described above.
-3. Don't change any of the Build & Development Settings.
-The default values will work fine.
-4. Press white "Deploy" button.
+3. In the Build & Development Settings,
+add `yarn run build` as the Build Command override.
```

Comment:
`yarn build` will do. Unlike `npm run build` the `run` in `yarn run build` is optional.

I feel `yarn build` is more elegant, so I'd suggest we use that?

Reply:
I will change the command to `yarn build`. (I only chose `yarn run build` because that was the command Vercel was previously using.)